### DISCUSSION TITLE
search ustack

### DISCUSSION THREAD

**Discussion by muttiopenbts** [bpftrace/bpftrace#2498] [General]
I want to look for the presence of a function name within ustack.
Is there a way to do something like
```
if (strcontains(ustack, "my_function"))
{
    print("found it");
}
```

**Comment by viktormalik** [comment]
This is not possible at the moment, we're only storing stack IDs in the BPF program. 

If there's a reasonable use-case for this, please open an issue so that we can track this feature request.
